### PR TITLE
added styles for image component

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added styles for image component - square, rounded & responsive image
Checkout the preview here -
https://user-images.githubusercontent.com/66695068/151985627-30659e1b-4888-40f9-8b74-7ebf5bc73ac0.mp4

### CODE REVIEW

**Review by sakleprajwal — COMMENTED** [review]
All good. Please go ahead.

File: stylesheets/components/image.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.img-round {
+  height: 9rem;
+  width: 9rem;
+  border-radius: 50%;
+  margin: 0.8rem;
+  object-fit: cover;
```
Comment (vivekmehto) [inline stylesheets/components/image.css]: you can give object-fit in reset CSS directly to the img tag.

File: stylesheets/components/image.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.img-sq {
+  height: 9rem;
+  width: 9rem;
+  margin: 0.8rem;
+  object-fit: cover;
```
Comment (vivekmehto) [inline stylesheets/components/image.css]: the same set of code is repeating.

**Review by vivekmehto — COMMENTED** [review]
Avoid same lines of code.

**Comment by aishwarya-bm** [comment]
made changes for common style, thanks!

**Pull request merged**